plutus-tx-plugin/test/Budget/9.2/show.budget.golden

```diff
@@ -1,2 +1,2 @@
-({cpu: 2926804219
-| mem: 8741204})
\ No newline at end of file
+({cpu: 2695424219
+| mem: 7735204})
```

Comment:
I didn't optimize any `show` logic specifically, this is just from the general optimizations of the stdlib code.